### PR TITLE
feat(rdb): shared responsibility MTA-4383

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+You remain responsible for configuring encryption for the SQL clients connecting to the database, by carrying out actions such as activating SSL in client configuration options or renewing TLS certificates.
+
+When used by SQL Clients, SSL connections will be performed end-to-end, i.e. from your SQL client up to the Database Instance executing SQL queries.
+
+<Message type="note">
+  Scaleway's components in between such as load balancers do not decrypt Managed Databases' traffic.
+</Message>
+
+### Scaleway Access
```
Comment (bene2k1) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:174]:
```suggestion
### Scaleway access
```

**Review by bene2k1 — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+When [promoting a Read Replica](/managed-databases/postgresql-and-mysql/how-to/manage-read-replica/#how-to-promote-a-read-replica) to a main node, make sure that the main node has completed all ongoing operations and that the Read Replica lag is down to zero before promoting it.
+
+If ongoing operations are not complete before the promote operation starts, some operations from the main node might not be contained in the newly promoted node.
+
+Refer to the [How to manage Read Replicas](/managed-databases/postgresql-and-mysql/how-to/manage-read-replica/#how-to-promote-a-read-replica) documentation page for more information.
+
+
+## Monitoring
+
+Monitoring is an essential pillar to ensure the of security and reliability of your services. The practice provide real-time insights into the performance, security, and resource allocation of your databases. Knowing the behaviors of databases in real-time allows you to proactively identify and resolve issues and optimize operational efficiency.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:78]:
```suggestion
Monitoring is an essential pillar to ensure the security and reliability of your services. The practice provides real-time insights into the performance, security, and resource allocation of your databases. Knowing the behaviors of databases in real-time allows you to proactively identify and resolve issues and optimize operational efficiency.
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+### Metrics and logs
+
+Database default logs and metrics are stored inside **Cockpit**.
+
+This includes:
+
+  - **Metrics**: CPU, memory, disk usage, replication lag and number of connections. Metrics are refreshed every minute.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:86]:
```suggestion
  - **Metrics**: CPU, memory, disk usage, replication lag, and number of connections. Metrics are refreshed every minute.
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+Refer to the [Dealing with disk_full mode in a Database Instance](/managed-databases/postgresql-and-mysql/troubleshooting/disk-full/) documentation page for more information.
+
+<Message type="note">
+  You are responsible for monitoring disk size growth and performing any required disk size increases.
+</Message>
+
+
+### Database performance
+Scaleway creates your Database Instance with a default configuration fine-tuned to optimize database performance for standard workloads, and to monitor and solve issues caused by hardware, operating systems, or file system level failures.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:120]:
```suggestion
Scaleway creates your Database Instance with a default configuration fine-tuned to optimize database performance for standard workloads and monitor and solve issues caused by hardware, operating systems, or file system level failures.
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+## Security best practices
+
+For optimal security, we recommend that you:
+
+1. Activate connection encryption and certificate validation in clients connecting to the database
+    - **For PostgreSQL clients**: set the ssl-mode option to `verify-full`
+    - **For MySQL clients**: enable the `VERIFY_IDENTITY` option
+
+2. Restrict external access to databases by limiting IPs able to connect to database.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:219]:
```suggestion
2. Restrict external access to databases by limiting IPs able to connect to the database.
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+## Failover
+
+If you use [High Availability](#availability-and-durability) as your Database Instance configuration type, whenever the primary node becomes unavailable, a failover will be performed automatically after the unavailability is detected.
+
+### Read Replicas
+
+A Read Replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database node and any changes made are replicated to the replica asynchronously.
+
+Creating, deleting and promoting a Read Replica are actions performed by you. Scaleway then automatically creates, deletes or promotes the Read Replica.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:63]:
```suggestion
Creating, deleting, and promoting a Read Replica are actions performed by you. Scaleway then automatically creates, deletes or promotes the Read Replica.
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+Refer to the [How to manage backups](/managed-databases/postgresql-and-mysql/how-to/manage-manual-backups/#how-to-restore-backups) and the [How to manage snapshots](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/#how-to-create-a-database-instance-from-a-snapshot) documentation pages for more information.
+
+## Failover
+
+If you use [High Availability](#availability-and-durability) as your Database Instance configuration type, whenever the primary node becomes unavailable, a failover will be performed automatically after the unavailability is detected.
+
+### Read Replicas
+
+A Read Replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database node and any changes made are replicated to the replica asynchronously.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:61]:
```suggestion
A Read Replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database node, and any changes made are replicated to the replica asynchronously.
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+          The `log_min_duration_statement` parameter is set at `5000` and cannot be changed.
+        </Message>
+
+        <Message type="tip">
+          For more granular log details, you can activate the [pgaudit extension](/managed-databases/postgresql-and-mysql/api-cli/pg-audit/).
+        </Message>
+      - MySQL logs at a default level `log_error_verbosity=2` ([see MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_log_error_verbosity)). Logging slow queries is not activated by default, but you can activate it using `slow_query_log` and `long_query_time`. 
+
+<Message type="important">
+  Any changes performed on the default log verbosity levels and threshold can lead to an increase in storage size which might make your database unavailable. These potential impacts on the database hence remain your responsibility.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:99]:
```suggestion
  Any changes performed on the default log verbosity levels and threshold can lead to an increase in storage size, which might make your database unavailable. These potential impacts on the database hence remain your responsibility.
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+
+### Encryption in transit
+Managed Databases for PostgreSQL and MySQL supports SSL connections and allows you to download TLS certificates from Managed Databases.
+
+You remain responsible for configuring encryption for the SQL clients connecting to the database, by carrying out actions such as activating SSL in client configuration options or renewing TLS certificates.
+
+When used by SQL Clients, SSL connections will be performed end-to-end, i.e. from your SQL client up to the Database Instance executing SQL queries.
+
+<Message type="note">
+  Scaleway's components in between such as load balancers do not decrypt Managed Databases' traffic.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:171]:
```suggestion
  Scaleway's components in between, such as load balancers, do not decrypt Managed Databases' traffic.
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+
+Managed Databases for PostgreSQL and  MySQL allow you to manage access to user accounts inside your database.
+
+These typically correspond to PostgreSQL or MySQL users inside your Database Instance who can connect to your database and perform SQL queries.
+
+To ensure your Database Instance security and reliability, we limit the set of SQL actions you can perform or give to other users, such as `CREATE PUBLICATION` or `CREATE USER user SUPERUSER` in PostgreSQL.
+
+You are responsible for giving access to the relevant users or applications and reviewing these accesses frequently.
+
+Moreover, these accesses are stored inside your Database Instance only and are independent from Scaleway IAM, which means you are responsible for ensuring they are properly backed up or removed when required.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:207]:
```suggestion
Moreover, these accesses are stored inside your Database Instance only and are independent of Scaleway IAM, which means you are responsible for ensuring they are properly backed up or removed when required.
```

**Review by jcirinosclwy — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 11]
```diff
@@ -0,0 +1,223 @@
...
+You can [modify some advanced settings](/managed-databases/postgresql-and-mysql/how-to/configure-advanced-settings/), such as `effective_cache_size` or `max_connections` in PostgreSQL, but you are responsible for any impacts on your database availability and performance.
+
+### Version management
+
+Two types of engine version upgrades are available for your databases:
+
+| Upgrade Type | Description|
+| ------------ | ------------------- |
+| Minor versions | These are updates within the same version and are performed automatically. The updates include security fixes and bug fixes and are designed to be backward compatible. |
+| Major versions    | You can trigger major version upgrades on your Database Instance. Once you activate upgrade, Scaleway automatically creates a clone of your database with the chosen PostgreSQL version. Depending on the option you choose, you might need to manually switch traffic to your newly created database and/or stop write operations on your source database. Refer to the [How to upgrade your Database Instance engine version](/managed-databases/postgresql-and-mysql/how-to/upgrade-version/) documentation page for more information|
```
Comment (nerda-codes) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:145]:
```suggestion
| Major versions    | You can trigger major version upgrades on your Database Instance. Once you trigger an upgrade, Scaleway automatically creates a clone of your database with the chosen PostgreSQL version. Depending on the option you choose, you might need to manually switch traffic to your newly created database and/or stop write operations on your source database. Refer to the [How to upgrade your Database Instance engine version](/managed-databases/postgresql-and-mysql/how-to/upgrade-version/) documentation page for more information|
```

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 12]
```diff
@@ -0,0 +1,223 @@
...
+
+### Encryption in transit
+Managed Databases for PostgreSQL and MySQL supports SSL connections and allows you to download TLS certificates from Managed Databases.
+
+You remain responsible for configuring encryption for the SQL clients connecting to the database, by carrying out actions such as activating SSL in client configuration options or renewing TLS certificates.
+
+When used by SQL Clients, SSL connections will be performed end-to-end, i.e. from your SQL client up to the Database Instance executing SQL queries.
+
+<Message type="note">
+  Scaleway's components in between, such as load balancers, do not decrypt Managed Databases' traffic.
```
Comment (nerda-codes) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:171]:
```suggestion
  Scaleway's components in between, such as Load Balancers, do not decrypt Managed Databases' traffic.
```

Comment (nerda-codes) [reply managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:171]: Shouldn't it be "Scaleway's in between components"? not sure about this but "Scaleway's components in between" sounds weird to me

Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:171]: I dont think so bc in this case it's talking about the specific occurrence and not characterizing these components as always in between, if that makes sense? but we can ask a native

Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:171]:
```suggestion
  Any intermediary Scaleway components, such as Load Balancers, do not decrypt Managed Databases' traffic.
```
thanks @RoRoJ

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 13]
```diff
@@ -0,0 +1,223 @@
...
+
+You are responsible for:
+    - Any impacts on performance incurred from SQL queries or database-specific configurations you set up, such as creating indexes, triggering backups, or changing advanced settings  (e.g. `max_connections`).
+    - Sizing the Database Instance appropriately for your workload.
+
+## Configuration and version management
+
+Configuration and version management are critical for maintaining consistency, reliability, and scalability across your services, and to ensure that deployments are accurately configured, tracked, and updated.
+
+This practice fosters streamlined development processes, minimizes errors, and facilitates efficient troubleshooting, ultimately enhancing overall system stability and agility.
```
Comment (fpagny) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:130]:
```suggestion

```
I would remove this line ("This practice (...) agility"), I find it too generic to provide significant information to a customer

File: managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx [hunk 14]
```diff
@@ -0,0 +1,223 @@
...
+| Minor versions | These are updates within the same version and are performed automatically. The updates include security fixes and bug fixes and are designed to be backward compatible. |
+| Major versions    | You can trigger major version upgrades on your Database Instance. Once you trigger an upgrade, Scaleway automatically creates a clone of your database with the chosen PostgreSQL version. Depending on the option you choose, you might need to manually switch traffic to your newly created database and/or stop write operations on your source database. Refer to the [How to upgrade your Database Instance engine version](/managed-databases/postgresql-and-mysql/how-to/upgrade-version/) documentation page for more information|
+
+### Database extensions
+
+For security reasons, we support a defined set of [databases extensions for PostgreSQL](/faq/databases-for-postgresql-and-mysql/#which-postgresql-extensions-are-available).
+
+For security and reliability reasons, we only support a specific version of each extension for each PostgreSQL major version.
+
+This means we will only update the versions of an extension that are available with our supported PostgreSQL major versions.
```
Comment (fpagny) [inline managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx:153]:
```suggestion
This means we will only update the available versions of an extension when supporting a new PostgreSQL major versions.
```

**Review by fpagny — APPROVED** [review]

**Pull request merged**